lib/memoryprofiler/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "wasmer-memoryprofiler"
```

Comment:
I think we should consider not including it in this repo and not naming it anything to do with Wasmer. It really is a standalone utility that could be generally useful and it shouldn't follow our versioning scheme for the rest of Wasmer.

1.0 implies stability and a new thing we just made is likely not 1.0-stable especially because we want to merge early and often into master and release whenever

Reply:
I absolutely agree with you, but I don't know where else to put it. A new repo in github? Or as a directory in wasmer/ which isn't part of the wasmer workspace?

Reply:
https://github.com/wasmerio/loupe